user-interface/src/main/java/life/qbic/datamanager/views/account/AccessTokenDeletionConfirmationNotification.java

```diff
@@ -0,0 +1,31 @@
+package life.qbic.datamanager.views.account;
+
+import com.vaadin.flow.component.Text;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import life.qbic.datamanager.views.notifications.NotificationDialog;
+
+/**
+ * Warns the user that the personal access token will be deleted and cannot be used
+ * <p>
+ * This dialog is to be shown when PAT delection is triggered by the user.
+ */
+public class AccessTokenDeletionConfirmationNotification extends NotificationDialog {
+
+  public AccessTokenDeletionConfirmationNotification() {
+    addClassName("batch-deletion-confirmation");
+    customizeHeader();
+    content.add(new Div(new Text(
```

Comment:
I'd suggest to employ a Span component here which can be filled with the text directly: